Gemfile

```diff
@@ -31,7 +31,7 @@ gem 'ros-apartment', require: 'apartment'
 gem 'ros-apartment-sidekiq', require: 'apartment-sidekiq'
 gem 'apartment-activejob'
 gem 'devise'
-gem "comfortable_mexican_sofa",git: 'https://github.com/restarone/comfortable-mexican-sofa', tag: '3.4'
+gem "comfortable_mexican_sofa",git: 'https://github.com/ajaystha42/comfortable-mexican-sofa', branch: 'feat/parser_modifications'
```

Comment:
@ajaystha42 this should be restarone/cms

Reply:
@donrestarone we made the changes in cms for this task, 
To test the changes in review app, we need to make this change before merging the changes in https://github.com/restarone/comfortable-mexican-sofa/pull/13
This change will be reverted once the UAT passes

Reply:
@ajaystha42 you need to point at restarone/cms., as I said before. We aren't deploying your fork, only our own

Reply:
@donrestarone  I pointed it to restarone/cms. But, the changes wont work until we merge [this](https://github.com/restarone/comfortable-mexican-sofa/pull/13)